src/config.ts

```
@@ -0,0 +1,40 @@
+class Pluggable {
+  installedPlugins: any
+  constructor() {
+    this.installedPlugins = []
+  }
+
+  install(handler, options = {}) {
+    if (typeof handler !== 'function') {
+      console.error('plugin.install must receive a function')
+      handler = () => ({})
```

Should decide if we're going to exit the process here or quietly swallow the misconfigured plugin

---

I think failing loudly is usually a good idea

---

The louder the better, ideally with "you screwed up and here's where and how to fix it"

---

Yeah. I'm fine with that. I wish I had a plugin name to point people at.

---

hmmm didn't think of that. 🤔